vNext/extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -266,6 +267,25 @@ export class AjaxMonitor implements ITelemetryPlugin, IDependenciesPlugin, IInst
                 dependency.correlationContext = /* dependency.target + " | " + */ correlationContext;
             }
 
+            if (this._config.enableHeaderTracking) {
```

Comment:
Can you also add request header tracking. For fetch, it is accessible via the `init?: RequestInit` argument. For xhr you can modify the `XMLHttpRequest.prototype.setRequestHeader` to store it also store it in `this` so that we can look it up later